### DISCUSSION TITLE
Screen Save & Restore

### DISCUSSION THREAD

**Discussion by WillianBR** [magiblot/tvision#89] [Q&A] [answered]
Does anybody manage to save the screen and restore it later?  I need a way to create a kind of draw app in TUI.  But I need to be able to save and restore the screen/desktop to memory.

If anybody is using this TV library to create tools, please call me. May be we can exchange knowledge.

**Comment by magiblot** [comment, marked as answer]
Hi Willian!

There's an official way to do this: streamable classes. However, I don't recommend anybody to use them.

Another person tried to do this, and they had to create a JSON file in order to read it later. You can find their project at https://github.com/IngvarRiga/TurboVision-dialog-designer.

**Comment by WillianBR** [comment]
Thank you!
